python/common/business-registry-model/tests/models/test_configuration.py

```diff
@@ -20,25 +20,26 @@
 
 from business_model.models import Configuration
 
-def test_valid_configuration_save(session):
-    """Assert that a valid configuration can be saved."""
-    configuration = Configuration(
-        name='NEW_TEST_CONFIGURATION',
-        val='100',
-    )
-    configuration.save()
-    assert configuration.id
+# TODO: Do we need to create configs on the fly?
```

Comment:
Yeah it seems these are only populated by migrations so probably not, but it does look like these would work if you set 'name' to a valid name enum value? I think since you updated the type from string to enum it fails because the enum doesn't contain 'NEW_TEST_CONFIGURATION'

Reply:
yep, hence my question, I couldn't see a place that didn't use the Enum